s30291-pj-StanislasHinsinger/zajecia1/instalacja_debian_linux.md

```diff
@@ -0,0 +1,361 @@
+
+> [!NOTE]
+> W procesie konfiguracji wirtualnej maszyny został stworzony dysk o pojemności 40GB. 4GB zostały przeznaczone dla przestrzeni wymiany, a dla reszty systemu według [minimalnych zalecen](https://www.debian.org/releases/stable/i386/ch03s04.en.html) w wersji `No Dektop`, czyli tej która jest instalowana, potrzebne jest minimum 4GB przestrzeni. Dla bezawaryjnego korzystania i pozostawienia wolnej przestrzeni dodano dodatkowo 6GB dla przyszłych aplikacji, zaokrąglając wymagana przestrzeń ogólna `/` do 10GB dla systemu, z tego względu na katalog `Home` można było przeznaczyć `26GB`.
+
+
+22. I właśnie w taki sposób prawidłowo została podzielona i skonfigurowana cala przestrzeń dyskowa. Aby zakończyć ten proces należy wybrać opcje `Finish partitioning and write changes to disk` a następnie `Yes`. Po tym zostanie rozpoczęty proces instalacji podstawy systemu.
+
+23. Po zainstalowaniu się podstawy systemu, na ekranie powinno wyskoczyć powiadomienie `Configure the package manager`, w którym mamy opcje zeskanować inne nośniki, które mogłyby zostać użyte na potrzeby `package manager'a`. W tym procesie instalacyjnym dodatkowe nośniki nie zostały wprowadzone, wiec należy wybrać opcje `No`. Po czym zostanie wyświetlone okno wyboru kraju z którego zostanie dokonany klon pobieranych plików systemu do finalnego procesu instalacyjnego. W tym przypadku należy wybrać `Poland`. Wyświetlone zostaną dostępne serwery archiwów systemowych, gdzie optymalnie należy wybrać serwer znajdujący się najbliżej lokalizacji instalowanego systemu. W przypadku Trójmiasta zalecane jest wybranie opcji `ftp.task.gda.pl`, jeżeli podczas instalacji wyskoczy błąd z informacja, ze serwer ten jest niedostępny lub  że nie można pobrać plików instalacyjnych, to w takim przypadku należy wybrać innych serwer dostępny z listy. Po wybraniu serwera, instalator zapyta o informacje czy do uzyskania dostępu do Internetu wykorzystany ma być serwer `Proxy`. Jeżeli sieć, w której instalowany jest system nie posiada ograniczaj dostępowych to należy zostawić tą przestrzeń pustą, w innym przypadku należy uzupełnić to pole danymi serwera Proxy.
+
+24. W trakcie pobierania plików instalacyjnych prawdopodobnie wyświetli się informacja odnośnie `ankiety popularnościowej`, w której nie trzeba brać udziału, dlatego tez w tym przypadku należy wybrać opcje `No`.
```

Comment:
Co to za ankieta?
Jakie informacje są zbierane przez tę ankietę?
Do czego są wykorzystywane dane z tej ankiety?